express-api/src/services/users/usersServices.ts

```diff
@@ -150,6 +151,19 @@ const addKeycloakUserOnHold = async (
     Note: note,
     CreatedById: systemUser.Id,
   });
+  const newAccessRequest: AccessRequestData = {
```

Comment:
Good catch @GrahamS-Quartech , when I merged the branch, I guess I missed this because I had made the changes on a totally different branch and didn't even see your changes. I'll revert this back.